backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Tests/DelegationsControllerTest.cs

```diff
@@ -115,6 +115,49 @@ public async Task Post_DeleteRules_Success()
             AssertionUtil.AssertEqual(expected, actual);
         }
 
+        /// <summary>
+        /// Test case: Calling the POST operation for DeleteRules to perform a valid deletion of resourceRegistryId
+        /// Expected: DeleteRules returns status code 201 and list of rules created match expected
+        /// 
+        /// Scenario:
+        /// Calling the POST operation for DeleteRules to perform a valid deletion
+        /// Input:
+        /// List of two one rule in one policy for deletion of the resource between for a single offeredby/coveredby combination resulting in a single policyfile beeing updated.
+        /// Expected Result:
+        /// Rules are deleted and returned with the CreatedSuccessfully flag set and rule ids
+        /// Success Criteria:
+        /// DeleteRules returns status code 201 and list of rules deleted to match expected
+        /// </summary>
+        [Fact]
+        public async Task Post_DeleteResourceRegistryRules_Success()
```

Comment:
should also add test for DeletePolicy

Reply:
Should also have negative tests? E.g.
- Delete request without valid resource must fail